src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java

```diff
@@ -38,24 +39,26 @@ public void reportMethodAnnotation(Class<? extends Annotation> annotation,
                                     CommandHandler handler =
                                             method.getAnnotation(CommandHandler.class);
                                     Command command = new Command(handler.commandName(),
-                                            handler.description(), handler.roles());
+                                                                  handler.description(), handler.roles());
                                     for (Class<?> paramClazz : method.getParameterTypes()) {
                                         if (paramClazz.isAnnotationPresent(ParsedEntity.class)) {
                                             Field[] fields = allImplicitFields(paramClazz);
                                             for (Field field : fields) {
                                                 if (field.isAnnotationPresent(Flag.class)) {
-                                                    disparse.parser.Flag parseableFlag = Utils
+                                                    CommandFlag flag = Utils
                                                             .createFlagFromAnnotation(field, field
                                                                     .getAnnotation(Flag.class));
-                                                    CommandRegistrar.registrar.register(command,
-                                                            parseableFlag);
+                                                    CommandRegistrar.REGISTRAR
+                                                        .register(command,
```

Comment:
similarly here with google-style; again, I could just be wrong and this is how it formatted it